CHANGES.md

```diff
@@ -1,10 +1,47 @@
+## edge-19.10.5
+
+This edge release adds support for integrating Linkerd's PKI with an external
```

Comment:
will everyone know what PKI means? is it worth spelling it out?

Reply:
I think people reading the Linkerd changelogs will probably be familiar with this acronym, but I'm happy to expand it.